asv.conf.json

```diff
@@ -40,7 +40,7 @@
 
     // The Pythons you'd like to test against.  If not provided, defaults
     // to the current version of Python used to run `asv`.
-    "pythons": ["2.7", "3.8"],
+    "pythons": ["3.8", "3.9"],
```

Comment:
Do we really want to test 2 versions of python here?   Perhaps should not provision and just run against the version `asv` is installed against.